apps/sql_data/tests/test_admin.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from django.contrib.admin.sites import AdminSite
+from faker import Faker
+from model_bakery import baker
+from rest_framework.test import APIRequestFactory, APITestCase
+
+from apps.common.tests.test_common import LoggedInMixin
+from apps.sql_data.admin import DataSourceVersionAdmin
+from apps.sql_data.models import DataSourceVersion, SQLDatabaseSource
+
+pytestmark = pytest.mark.django_db
+
+fake = Faker()
+
+
+class InitializeTestData(LoggedInMixin, APITestCase):
```

Comment:
As stated above, use a more descriptive name and also consider inheriting from a more appropriate base class.